iOS/ViewController.m

```diff
@@ -274,6 +275,10 @@ - (void)viewDidLoad
     [self helpPopCheckSetUp];
     self.helpPopView.hidden = YES;
 
+    [self.placeButton setBackgroundImage:[[UIImage imageNamed:@"traceroute-button"] resizableImageWithCapInsets:UIEdgeInsetsMake(0, 22, 0, 22)] forState:UIControlStateNormal];
```

Comment:
I actually removed this in my layout changes on purpose, I switched to just doing it with cornerRadius and a background colour instead of the image, so this is unintentionally it back in.

Reply:
D'oh! Merge conflict resolution gone wrong. Fixing...